src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleList } from '../PeopleList/PeopleList';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoader, setIsLoader] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const fetchPeople = useCallback(async () => {
+    try {
+      setIsLoader(true);
+      const peopleFromServer = await getPeople();
+
+      setPeople(peopleFromServer);
+    } catch (error) {
+      setIsError(true);
+    } finally {
+      setIsLoader(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    fetchPeople();
+  }, []);
+
+  const { slug = '' } = useParams();
```

Comment:
move to the top of the component

Reply:
general structure should looks like that
1. navigation, styles,states, custom hooks, ..so on
2. useEffects
3. our component methods